journald/lib_journald.cpp

```diff
@@ -225,14 +251,16 @@ class JournaldContainerLoggerProcess :
     // level containers) , and the executor is named, use that name to
```

Comment:
Not yours but could you please kill that extra space before the comma?

Reply:
Done.